lib/voice/VoiceConnection.js

```diff
@@ -221,11 +242,16 @@ class VoiceConnection extends EventEmitter {
       if (this.listeners("debug").length > 0) {
         this.emit("debug", "Rec: " + JSON.stringify(packet));
       }
+      if (packet.seq) {
+        this.wsSequence = packet.seq;
+      }
       switch (packet.op) {
         case VoiceOPCodes.READY: {
           this.ssrc = packet.d.ssrc;
-          this.sendNonce.writeUInt32BE(this.ssrc, 8);
-          if (!packet.d.modes.includes(ENCRYPTION_MODE)) {
+          this.sendHeader.writeUInt32BE(this.ssrc, 8);
+
+          const availableMode = SUPPORTED_ENCRYPTION_MODES.find((mode) => packet.d.modes.includes(mode) && (mode === "aead_aes256_gcm_rtpsize" ? aes256Available : true));
```

Comment:
Not a big deal but I think we can simplify this check:
```suggestion
          const availableMode = SUPPORTED_ENCRYPTION_MODES.find((mode) => packet.d.modes.includes(mode) && (mode !== "aead_aes256_gcm_rtpsize" || aes256Available));
```